packages/rancher-logging/generated-changes/overlay/templates/_generic_logging.yaml

```diff
@@ -65,11 +65,9 @@ spec:
     configReloaderImage:
       repository: {{ template "system_default_registry" . }}{{ .Values.images.config_reloader.repository }}
       tag: {{ .Values.images.config_reloader.tag }}
-    {{- if not .Values.disablePvc }}
```

Comment:
The original implementer of this logic was Dan Bason afaics, did you ask him why it was implemented this way?

Reply:
Hey @dbason, Because of the `if` check you see above we're not able to configure the `fluentd.bufferStorageVolume` from chart values since we need `disablePvc`  set to true. I'm not aware of the reason why this check was implemented at that time. Can you please let us know why it was implemented that way?

Reply:
It was some time ago, so off the top of my head I don't remember the specifics sorry.  Out of interest how are you creating a volume for the buffers without a PVC?

Reply:
@superseb Since we have a requirement to add the ability to configure `fluentd.bufferStorageVolume` block via chart values on the logging resource and as @dbason confirmed it would be fine to remove the checks which are preventing us to do so, Should we go ahead with this change?